src/qt/diagnosticsdialog.cpp

```diff
@@ -34,6 +35,34 @@ DiagnosticsDialog::DiagnosticsDialog(QWidget *parent, ResearcherModel* researche
     GRC::ScaleFontPointSize(ui->diagnosticsLabel, 14);
     GRC::ScaleFontPointSize(ui->overallResultLabel, 12);
     GRC::ScaleFontPointSize(ui->overallResultResultLabel, 12);
+
+    //Construct the teests needed.
```

Comment:
"teests" should be "tests"

Reply:
Done

Reply:
Not done. Still teests... should be tests.